move_base:

  base_global_planner: global_planner/GlobalPlanner
  base_local_planner: base_local_planner/TrajectoryPlannerROS
  
  # Frequency parameters
  planner_frequency: 1.0
  controller_frequency: 5.0
  
  # Tolerance parameters
  planner_patience: 10.0
  controller_patience: 20.0
  
  # Recovery
  recovery_behavior_enabled: true
  clearing_rotation_allowed: true
  conservative_reset_dist: 3.0
  
  # Global Planner (alternative way to load params)
  global_planner:
    $(rosparam command="load" file="$(find autobot_ai)/config/global_planner.yaml")
    layers: ['static_layer', 'obstacle_layer', 'inflation_layer', 'semantic_layer']
    semantic_layer:
        enabled: true
        topic: /semantic_costmap